src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Model/UserAuthenticationResult.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+
+namespace Altinn.Platform.Authentication.Model
+{
+    /// <summary>
+    /// Handles UserProfile
+    /// </summary>
+    public class UserAuthenticationResult
+    {
+        /// <summary>
+        /// Gets or sets Identifier used to uniquely identify User
+        /// </summary>
+        public int UserID { get; set; }
+
+        /// <summary>
+        /// Gets or sets Identifier used to uniquely identify PartyID for user
```

Comment:
check comment

Reply:
Thank you! This was very wrong, I have corrected it in the latest commit.